File: samples/src/main/java/com/commit451/resourcespoet/sample/State.java
package com.commit451.resourcespoet.sample;

import com.squareup.moshi.Json;

/**
 * Defines a state from json
 */
public class State {

    @Json(name = "name")
    public String name;
    @Json(name = "abbreviation")
    public String abbreviation;

    private State() {

    }
}

File: samples/src/main/java/com/commit451/resourcespoet/sample/State.kt
package com.commit451.resourcespoet.sample

import com.squareup.moshi.Json

/**
 * Defines a state from json
 */
class State {

    @Json(name = "name")
    var name: String? = null
    @Json(name = "abbreviation")
    var abbreviation: String? = null
}
